timer/src/activities.ts

```diff
@@ -0,0 +1,25 @@
+// @@@SNIPSTART nodejs-timer-reminder-activity
+import { Context } from '@temporalio/activity';
+import mailgun from 'mailgun-js';
+
+const apiKey: string = process.env.MAILGUN_API ?? '';
+const domain: string = process.env.MAILGUN_DOMAIN ?? '';
+const to: string = process.env.ADMIN_EMAIL ?? '';
```

Comment:
I would still really like this sample to direct devs to use DI for injecting clients into activities instead of initializing them in the top level for better testability.
One other issue with these lines is that there's no assertion that all required env vars are provided, this is not good practice, prefer to fail fast.